core/state/node/transitions.go

```diff
@@ -53,6 +54,10 @@ func (t *InitalizationTransition) Patch(oldState []byte) ([]byte, error) {
 	}]`, marshaled)), nil
 }
 
+func (t *InitalizationTransition) Enrich(oldState []byte) error {
+	return nil
```

Comment:
should this instead return error if it's unexpected for this to ever be called on init?

Reply:
This is just a no-op, I can comment that as well.